README.md

```diff
@@ -26,4 +26,22 @@ about it at the start of each recursive call.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
-your answer to this markdown file.
+your answer to this markdown file.  
+  
+
+ANSWER:  
+I believe that a good invariant for my recursive implementation of fib() is that  
+at the beginning of every recursive call, the length of the array holding the  
+fibonacci values is exactly one more than the desired term in the fibonacci sequence.  
+In other words, if we were to call fib() on 7, then the array would have a length of 8.  
+The exception to this, of course, if we are dealing with negative numbers, in which case  
+the length of the list is always just 1 and will have a single entry of 0. 
+Using this relationship, we are able to always know exactly which index of the array  
+corresponds to which value in the fibonacci sequence. This can be used to prove that  
+we are always assigning the correct values to the correct places in the array.  
```

Comment:
Does this invariant help you to prove the algorithm correct? It doesn't say anything about what the values are, just how many there are.